build_projects/dotnet-host-build/build.ps1

```diff
@@ -17,7 +20,10 @@ if($Help)
     Write-Host ""
     Write-Host "Options:"
     Write-Host "  -Configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
-    Write-Host "  -Architecture  <ARCHITECTURE>      Build the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
+    Write-Host "  -Architecture  <ARCHITECTURE>      Build on the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
```

Comment:
>Is this script used only within Core-setup build? 

I am operating under that assumption, but I made it backwards compatible regardless. If all they set is the `Architecture` then they'll get the same behavior they had before since we set `TargetArch = $Architecture` when `TargetArch` is undefined. 

Which is nice because I didn't have to modify the netci.groovy file since the args to pass to build.cmd are unchanged.